qunicorn_core/core/pilotmanager/aws_pilot.py

```diff
@@ -35,21 +36,25 @@
 from qunicorn_core.static.enums.provider_name import ProviderName
 from qunicorn_core.static.enums.result_type import ResultType
 from qunicorn_core.util import logging
+from braket.ir.openqasm import Program
 
 
 class AWSPilot(Pilot):
     """The AWS Pilot"""
 
     provider_name: ProviderName = ProviderName.AWS
 
-    supported_language: AssemblerLanguage = AssemblerLanguage.BRAKET
+    supported_languages: list[AssemblerLanguage] = [AssemblerLanguage.BRAKET, AssemblerLanguage.QASM3]
 
     def run(self, job_core_dto: JobCoreDto):
         """Execute the job on a local simulator and saves results in the database"""
         if not job_core_dto.executed_on.is_local:
             raise return_exception_and_update_job(job_core_dto.id, ValueError("Device need to be local for AWS"))
 
         device = LocalSimulator()
+        for index in range(len(job_core_dto.transpiled_circuits)):
```

Comment:
Hier lieber for each anstatt for i in range: "for circuit in job_core_dto.transpiled_circuits" Nutzt den index ja nie als zahl

Reply:
doch wird als zahl verwendet damit wenn man den circuit wieder einträgt in die liste der richtige index ersetzt wird sonst müsste man ne 2. liste haben